docs/Configuring.md

```diff
@@ -523,3 +523,19 @@ system.workflow-heartbeats {
 ```
 
 The default threshold value is 100, just like the default for the heartbeat batch size.
+
+### Yaml
+
+Cromwell will throw an error when detecting cyclic loops in Yaml inputs. However one can craft small acyclic Yaml
```

Comment:
is `Yaml` worth capitalizing?

Reply:
In docs perhaps but not identifiers or config please